A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Engine.js

```diff
@@ -63,6 +63,10 @@ var A320_Neo_LowerECAM_Engine;
                     ignLeftTargetState = A320_Neo_LowerECAM_Engine.Definitions.IGN_STATE.A;
                 }
             }
+            if (n2Percent > 50) {
+                // Close left valve
+                this.engineLeft.setEngineBleedValveState(false, false);
+            }
```

Comment:
No need for seperate args, just put it all into 1 if statement.
```suggestion
            if (n2Percent > 50) {
                // Close left & right valve
                this.engineLeft.setEngineBleedValveState(false, false);
                this.engineRight.setEngineBleedValveState(false, false);
            }
```

Reply:
I don't think you can do this since n2Percent refers to the left engine at line 58, but is then subsequently redefined as the value for the right engine on line 73.

Reply:
Ahh yes.. In that case I will suggest something else.